src/controls/dimension-slider.js

```diff
@@ -311,6 +308,15 @@ export default class DimensionSlider extends EventSubscriber {
     }
 
     /**
+     * Projects along z or t
+     *
+     * @memberof DimensionSlider
+     */
+    projection(value) {
+        console.log(value);
+    }
```

Comment:
one left-over from @jburel's original PR. I think we should take out the entire method for now.